tox.ini

```diff
@@ -39,13 +41,22 @@ commands =
     python -c 'import pathlib; print("documentation available under file://\{0\}".format(pathlib.Path(r"{toxworkdir}") / "docs_out" / "index.html"))'
 
 # Distro
-[testenv:dist]
+[testenv:dist-{win, linux}]
 skip_install = true
-# Have to use /bin/bash or the `*` will cause that argument to get quoted by the tox command line...
+platform =
+    win: win|msys
+    linux: linux|darwin
+allowlist_externals = 
+    win: powershell
+    linux: bash  
```

Comment:
Could you explain why we need this?

Reply:
The wildcard usage like `nbclient*.whl` is quite difficult to get into windows, even the gh windows runners have the bash installed. It's all about what is replaced within the quotes. So, what I did here was to distangle Windows and POSIX like systems. `tox `will check the platform via `sys.platform` and select the correct environment. Via that it also runs, e.g., on Windows 10 systems, where there is not a bash installed in general.

It might be a bit blown up to get this one `whl` file into `pip install` but I haven't found another way.